src/main/java/core/basesyntax/figures/Figure.java

```diff
@@ -0,0 +1,29 @@
+package core.basesyntax.figures;
+
+public class Figure implements Area, UnitInfo {
+    private String color;
+    private String name;
+
+    public Figure(String color, String name) {
+        this.color = color;
+        this.name = name;
+    }
+
+    public String getColor() {
+        return color;
+    }
+
+    public String getName() {
+        return name;
+    }
+
+    @Override
+  public double getArea() {
+        return 0;
+    }
+
+    @Override
+      public String getUnitsInfo() {
+        return "";
```

Comment:
The `getUnitsInfo` method returns an empty string. Ensure this is appropriate for your use case or consider making `Figure` an abstract class.